salt/pillar/__init__.py

```diff
@@ -302,6 +298,119 @@ def __init__(self, opts, grains, minion_id, saltenv, ext=None, functions=None,
             else:
                 log.error('Pillar data must be a dictionary')
 
+    @property
```

Comment:
Sorry about that - I wasn't sure if you missed the significance WRT the overrides.